hubble/src/chain_id_query.rs

```diff
@@ -14,7 +17,18 @@ use unionlabs::{
     traits::ClientState,
     WasmClientType,
 };
+sol! {
+    contract IbcHandler {
+        function CreateClient(MsgCreateClient calldata) returns (string memory);
+    }
 
+    struct MsgCreateClient {
+        string client_type;
+        bytes client_state_bytes;
+        bytes consensusStateBytes;
+        address relayer;
+    }
+}
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/unionlabs/union/pull/2693#issuecomment-2293655644>#ef6b49</a><span> - 08/16/2024, 03:05 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The field names in the 'MsgCreateClient' struct do not follow Rust's naming conventions. Rust conventionally uses snake_case for field names, but the fields here use camelCase.
</div>
<div id="fix">
<b>Fix:</b> Rename the fields in the 'MsgCreateClient' struct to follow Rust's snake_case naming convention.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -25,7 +25,7 @@
      struct MsgCreateClient {
          string client_type;
          bytes client_state_bytes;
 -        bytes consensusStateBytes;
 +        bytes consensus_state_bytes;
          address relayer;
      }
  }
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged